src/ucp/am/eager.c

```diff
@@ -81,14 +81,14 @@ ucp_proto_eager_short_init(const ucp_proto_init_params_t *init_params)
         .super.cfg_priority  = 0,
         .super.min_length    = 0,
         .super.max_length    = SIZE_MAX,
+        .super.min_iov       = 3,
```

Comment:
👍

Reply:
maybe add comment to explain why it's 3?